wire-ios/Wire-iOS/Sources/UserInterface/StartUI/Common/EmptySearchResultsView.swift

```diff
@@ -240,3 +237,35 @@ final class EmptySearchResultsView: UIView {
         }
     }
 }
+
+// MARK: - Previews
+
+struct EmptySearchResultsViewRepresentable: UIViewRepresentable {
+    typealias UIViewType = EmptySearchResultsView
+
+    var isSelfUserAdmin: Bool
+    var isFederationEnabled: Bool
+    fileprivate var initialState: EmptySearchResultsViewState
+
+    func makeUIView(context: Context) -> EmptySearchResultsView {
+        let view = EmptySearchResultsView(
+            isSelfUserAdmin: isSelfUserAdmin,
+            isFederationEnabled: isFederationEnabled
+        )
+        view.updateStatus(searchingForServices: false, hasFilter: false)
+        view.state = initialState
+        return view
+    }
+
+    func updateUIView(_ uiView: EmptySearchResultsView, context: Context) {
+        // You can update the view with new parameters here if needed
+    }
+}
+
+#Preview {
+    EmptySearchResultsViewRepresentable(
+        isSelfUserAdmin: true,
+        isFederationEnabled: false,
+        initialState: .noServicesEnabled
+    )
+}
```

Comment:
can't we use #Preview directly with EmptySearchResultsView ?

Reply:
Nope, because it's available from iOS 17 and above.

Reply:
Maybe use `if #available(iOS 17.0)` already?

Reply:
I decided to simplify it even more and have previews on iOS 17 and above since our most common simulator on a day to day basis is iOS 17 anyway.